<launch>

  <rosparam file="$(find slaw_arm_navigation)/config/arm_joint_trajectory_action.yaml" command="load" ns="arm_1" />
  <!-- <rosparam file="$(find slaw_arm_navigation)/config/gripper_joint_trajectory_action.yaml" command="load" ns="arm_1" /> -->

  <!-- <group ns="arm_1/gripper_controller"> -->
  <!--   <node pkg="slaw_arm_navigation" type="gripper_trajectory_action.py" name="gripper_trajectory_action" output="screen"> -->
  <!--     <remap from="joint_states" to="/joint_states"/> -->
  <!--   </node> -->
  <!-- </group> -->
  
  <group ns="arm_1/arm_controller">
    <node pkg="slaw_arm_navigation" type="joint_trajectory_action.py" name="joint_trajectory_action" output="screen">
      <remap from="joint_states" to="/joint_states"/>
    </node>
  </group>

</launch>